R/run-source.r

```diff
@@ -137,8 +137,8 @@ find_gist <- function(id, filename) {
     }
 
   } else {
-    if (length(r_files) > 1) {
-      warning("Multiple R files in gist, using first.")
+    if (length(r_files) >= 1) {
```

Comment:
This is rather confusing logic. I think it would be better as:

```
if (length(r_files) == 1) {
   ...
} else {
   ...
}
```

Presumably zero files should also be an error.